test/server/admin/stats_handler_test.cc

```diff
@@ -629,12 +610,7 @@ TEST_P(AdminStatsTest, UsedOnlyStatsAsJsonFilterString) {
   h3.recordValue(100);
 
   store_->mergeHistograms([]() -> void {});
-
-  std::map<std::string, uint64_t> all_stats;
-  std::map<std::string, std::string> all_text_readouts;
-  std::string actual_json =
-      statsAsJsonHandler(all_stats, all_text_readouts, store_->histograms(), true,
-                         absl::optional<std::regex>{std::regex("h[12]")});
+  std::string actual_json = statsAsJsonHandler(/*store_->histograms(),*/ true, "h[12]").second;
```

Comment:
why is `store_->histograms()` commented out here in some calls to this, but not in others?

Reply:
also nit: const actual_json?, here and elsewhere